MovieAppPetProject/MovieAppPetProject/HomeScreen/NewHomeViewController.swift

```diff
@@ -40,16 +49,24 @@ class NewHomeViewController: UIViewController, UITableViewDelegate, UITableViewD
         return 1
     }
 
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 190
+    }
+
     func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
-        guard let cell = tableView.dequeueReusableCell(withIdentifier: "reuseTableCell", for: indexPath) as? NewTableViewCell else {
-            fatalError("Unable to dequeue NewTableViewCell.")
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: Constants.Identifiers.reuseTableCell, for: indexPath) as? NewTableViewCell else {
+            return fetchDefaultTableCell()
```

Comment:
You can omit this abstraction. It's unnecessary.😄 You can just say else return UITableViewCell() 😄
Abstractions are useful but you must be careful when to use them!😄